src/fireflyClient.ts

```diff
@@ -233,6 +239,36 @@ export class FireflyClient {
     return true;
   }
 
+  /**
+   * Funds gas tokens to user's account
+   * @returns Fund gas reponse
+   */
+  async fundGas() {
+    const token = await this.getToken()
+    const headers: AxiosRequestHeaders = {
+      "Authorization": `Bearer ${token}`
+    }
+    const configs: AxiosRequestConfig = {
+      headers: headers
+    }
+
+    const response = await this.apiService.post<FundGasResponse>(
+      SERVICE_URLS.USER.FUND_GAS,
+      {},
+      configs
+    );
+    return response;
+  }
+
+  /**
+   * Returns boba balance in user's account
+   * @returns Number representing boba balance in account
+   */
+  async getBobaBalance() {
+    const balance = await this.getWallet().getBalance()
```

Comment:
no need of making this const balance variable, the entire method could be just a single liner